UnityProject/Assets/Scripts/Core/Chat/ChatRelay.cs

```diff
@@ -86,21 +92,50 @@ public void PropagateChatToClients(ChatEvent chatEvent)
 				//Send chat to PlayerChatLocation pos, usually just the player object but for AI is its vessel
 				var playerPosition = players[i].Script.PlayerChatLocation.OrNull()?.AssumedWorldPosServer()
 					?? players[i].Script.gameObject.AssumedWorldPosServer();
-
-				if (Vector2.Distance(chatEvent.position, playerPosition) > 14f)
+					
+				//Do player position to originator distance check
+				if (DistanceCheck(playerPosition) == false)
 				{
-					//Player in the list is too far away for local chat, remove them:
+					//Distance check failed so if we are Ai, then try send action and combat messages to their camera location
+					//as well as if possible
+					if (chatEvent.channels.HasFlag(ChatChannel.Local) == false &&
+					    players[i].Script.PlayerState == PlayerScript.PlayerStates.Ai &&
+					    players[i].Script.TryGetComponent<AiPlayer>(out var aiPlayer) &&
+					    aiPlayer.IsCarded == false)
+					{
+						playerPosition = players[i].Script.gameObject.AssumedWorldPosServer();
```

Comment:
shouldn't everyone use AssumedWorldPosition?

Reply:
they'll use and check it at line 93, but Ai uses its core there. The core isnt actually the player for the Ai, so line 106, checks the player for Ai